### DISCUSSION TITLE
why api/swagger.json miss?

### DISCUSSION THREAD

**Discussion by chenjianquan7** [DependencyTrack/dependency-track#3140] [Q&A]
### Current Behavior

![image](https://github.com/DependencyTrack/dependency-track/assets/26828709/2914cc22-a7e6-410e-8223-e3aa95c3c156)

i am put an sbom by rest api,it return 405,
when i look for api define, it is miss

### Steps to Reproduce

1.

### Expected Behavior

return html

### Dependency-Track Version

4.7.x

### Dependency-Track Distribution

Container Image

### Database Server

PostgreSQL

### Browser

Google Chrome

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this defect was already reported

**Comment by nscuro** [comment]
You're requesting the frontend (`http://localhost:8080`), the API server is running on `http://localhost:8081`, given you used the default `docker-compose.yml` file.
